app/pages/admin/project-status.jsx

```diff
@@ -108,19 +113,14 @@ class ProjectStatus extends Component {
   handleToggle(event, workflow) {
     this.setState({ error: null });
     const isChecked = event.target.checked;
-    let defaultWorkflowId;
-
-    if (this.state.project.configuration) {
-      defaultWorkflowId = this.state.project.configuration.default_workflow;
-    }
 
-    if (defaultWorkflowId === workflow.id && workflow.active) {
+    if (this.state.defaultWorkflowId === workflow.id && workflow.active) {
       this.setState({
         dialogIsOpen: true
       });
     }
 
-    if ((defaultWorkflowId !== workflow.id) || (defaultWorkflowId === workflow.id && !workflow.active)) {
+    if ((this.state.defaultWorkflowId !== workflow.id) || (this.state.defaultWorkflowId === workflow.id && !workflow.active)) {
```

Comment:
As I was updating this conditional, I was wondering if it's necessary - I tried changing it to just be an `else` attached to the conditional above it, and the UI functionality seems to be the same (tests all still passing as well). 

Would appreciate input on whether it's necessary here to explicitly check for the opposite of the conditions described [above](https://github.com/zooniverse/Panoptes-Front-End/blob/2a53e8b2172c3ed39eaa5f9847839131b1c2ba20/app/pages/admin/project-status.jsx#L123).